src/identity/mod.rs

```diff
@@ -32,6 +32,14 @@ pub enum PublicId {
     App(app::PublicId),
 }
 
+/// An enum representing the Full Id variants for a Client or App
+pub enum FullId {
```

Comment:
Do we need this in safe-nd?  I don't think it's something vaults will ever need.  I'm not really able to even see a use case for it in SCL, but I'm not too familiar with that crate.

Reply:
@Fraser999 I needed this wrapper so that I can add it to the mock [routing struct here](https://github.com/lionel1704/safe_client_libs/blob/78c2c063d69eacff185d4e1a685a5fc2174bc6ae/safe_core/src/client/mock/routing.rs#L86). Maybe once mock routing is deprecated we wouldn't need this.